### ISSUE TITLE
Bug 58075193: [React Native Gallery -> Sketch]: Upon selecting any color, screen reader fails to announce the selected state information.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#618]
Build Details:
OS Version: Dev (OS Build 27871.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.79.0

AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Media' button present in left navigation pane and expand it.
Navigate to 'Sketch' button and activate it.
Navigate to any color and activate it.
Listen to narrator announcement.
Actual Result:
Upon selecting any color, screen reader fails to announce the selected state information.
While navigating in between the different colors also, narrator is not announcing the selected state of the colors. It simply announces as "Black, button", or "Red, button" etc.

Note: Same issue is observed in 'Text-to-Speech' control also.

Expected Result:
Narrator is should the selected state of the color. When user navigate in between the colors narrator should announce as "Black, button, Selected", etc.

User Impact:
Users with visual impairments who rely on screen reader will not be able to know whether the button is selected/unselected if narrator fails to announce the state of the buttons. 

MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

Trap ID: 
[1.6 - Poor Grouping](https://aka.ms/PoorGrouping)

"Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by vineethkuttan** [comment]
This belongs to Community Module react-native-sketch-canvas. Which won't be the part of Fabric anytime soon.
